install-steps.md

```diff
@@ -1,193 +1,821 @@
+export cmd=openshift-baremetal-install
+export pullsecret_file=~/pull-secret.txt
+export extract_dir=$(pwd)
+# Get the oc binary
+curl -s https://mirror.openshift.com/pub/openshift-v4/clients/ocp-dev-preview/$VERSION/openshift-client-linux-$VERSION.tar.gz | tar zxvf - oc
+# Extract the baremetal installer
+./oc adm release extract --registry-config "${pullsecret_file}" --command=$cmd --to "${extract_dir}" ${RELEASE_IMAGE}
+~~~
+
+## Configure the install-config and run the installer
+
+1. Configure the `install-config.yaml` (Make sure you change the pullSecret and sshKey)
+    ~~~yaml
+    apiVersion: v1
+    baseDomain: <domain>
+    metadata:
+      name: <cluster-name>
+    networking:
+      machineCIDR: <public-cidr>
+    compute:
+    - name: worker
+      replicas: 2
+    controlPlane:
+      name: master
+      replicas: 3
+      platform:
+        baremetal: {}
+    platform:
+      baremetal:
+        apiVIP: <api-ip>
+        ingressVIP: <wildcard-ip>
+        dnsVIP: <dns-ip>
+        provisioningBridge: provisioning
+        externalBridge: baremetal
+        hosts:
+          - name: openshift-master-0
+            role: master
+            bmc:
+              address: ipmi://<out-of-band-ip>
+              username: <user>
+              password: <password>
+            bootMACAddress: <NIC1-mac-address>
+            hardwareProfile: default
+          - name: openshift-master-1
+            role: master
+            bmc:
+              address: ipmi://<out-of-band-ip>
+              username: <user>
+              password: <password>
+            bootMACAddress: <NIC1-mac-address>
+            hardwareProfile: default
+          - name: openshift-master-2
+            role: master
+            bmc:
+              address: ipmi://<out-of-band-ip>
+              username: <user>
+              password: <password>
+            bootMACAddress: <NIC1-mac-address
+            hardwareProfile: default
+          - name: openshift-worker-0
+            role: worker
+            bmc:
+              address: ipmi://<out-of-band-ip>
+              username: <user>
+              password: <password>
+            bootMACAddress: <NIC1-mac-address
+            hardwareProfile: default
+          - name: openshift-worker-1
+            role: worker
+            bmc:
+              address: ipmi://<out-of-band-ip>
+              username: <user>
+              password: <password>
+            bootMACAddress: <NIC1-mac-address
+            hardwareProfile: default
+    pullSecret: '<pull_secret>'
+    sshKey: '<ssh_pub_key>'
+    ~~~
+
+NOTE: Ensure to change the appropriate variables to match your environment.
+
+2. Create a directory to store cluster configs
+    ~~~sh
+    mkdir ~/clusterconfigs
+    cp install-config.yaml ~/clusterconfigs
+    ~~~
+
+3. Ensure all baremetal nodes are powered off prior to installing the OpenShift cluster
+   ~~~sh
+   ipmitool -I lanplus -U <user> -P <password> -H <management-server-ip> -p 6230 power off
```

Comment:
When I do this command I dont require port 6230. @schmaustech @e-minguez @karmab - I believe this isn't needed. Command works without.

Reply:
since no comment and I was able to confirm it worked without I'm resolving.